# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/call/video_receive_stream.h
# Commit: 0fc8c65e163e
# Full Hash: 0fc8c65e163ed5ccfc09c55ec906973cf35a79b4
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 3
# Description:
#   Bug 1376873 - Use audio/video sync groups; r=bwc
#   
#   This uses sync groups in the receive stream configs for the conduits rather
#   than establishing sync through direct calls. When the streams are created
#   in call.cc, ConfigureSync is called, which results in SetSync being called
# ==============================================================================

diff -r de3f5bd6846b -r 0fc8c65e163e media/webrtc/trunk/webrtc/call/video_receive_stream.h
--- a/media/webrtc/trunk/webrtc/call/video_receive_stream.h	Wed Feb 07 15:00:17 2018 -0500
+++ b/media/webrtc/trunk/webrtc/call/video_receive_stream.h	Fri Apr 27 07:25:19 2018 -0400
@@ -238,9 +238,6 @@
   // TODO(pbos): Add info on currently-received codec to Stats.
   virtual Stats GetStats() const = 0;
 
-  //TODO: find replacement for this using call interface
-  //virtual void SetSyncChannel(VoiceEngine* voice_engine, int audio_channel_id) = 0;
-
   // Takes ownership of the file, is responsible for closing it later.
   // Calling this method will close and finalize any current log.
   // Giving rtc::kInvalidPlatformFileValue disables logging.
